docs/native/src/wopi_requirements.rst

```diff
@@ -5,7 +5,7 @@ WOPI implementation requirements for |Office iOS| integration
 =============================================================
```

Comment:
I think my biggest problem with this section as a whole is that it _implies_ hosts don't have to implement operations that Office for iOS doesn't currently call. But that's not accurate. If we're expecting partners to implement everything (and I think we are), then listing out what operations are actually called today doesn't really serve a purpose IMO.